bulk.yml

```diff
@@ -2,10 +2,12 @@
 continue: "true"
 transfer: "ln_s"
 exclude: "clientpath"
-checksum_algorithm: "File-Size-64"
-skip: "minmax"
+skip: "all"
+parallel_upload: "10"
+parallel_fileset: "10"
 logprefix: "logs/"
 output: "yaml"
+l: readers.txt
```

Comment:
Once we have a build with https://github.com/openmicroscopy/openmicroscopy/pull/5849, this can even become

```
reader: readers.txt
```